curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b0bac4577060412809a6c.md

```diff
@@ -0,0 +1,97 @@
+---
+id: 646b0bac4577060412809a6c
+title: Step 15
+challengeType: 0
+dashedName: step-15
+---
+
+# --description--
+
+Create a `div` with a class name `dropdown-container` that will be used to hold the dropdown menu.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed